moveit_ros/planning/planning_scene_monitor/src/planning_scene_monitor.cpp

```diff
@@ -50,6 +50,11 @@
 
 #include <memory>
 
+namespace
+{
+constexpr double DEFAULT_PLANNING_SCENE_PUBLISH_FREQUENCY = 30.0; // Hz
+}  // namespace
+
```

Comment:
Hm, I remember we had long debates about this in some other thread before (@felixvd and @simonschmeisser were involved I think?)
What was the consensus here?

30Hz strikes me as hell of a lot when octomaps or many objects might be involved...

Reply:
yes, I proposed that before. It will only create issues if the octomap or objects change frequently as well. Which could be solved with input rate limiting in case of the octomap. I then tried to bargain for a few more FPS but I was told we're not on a bazaar here :-P

Just yesterday I stumbled over a video from IronOx and marveled at the tasty fresh green lettuce when I suddenly saw a PlanningSceneDisplay updating at 4fps and cried a little ;-)

Reply:
I agree that we should increase that value, but it seems @felixvd found some issues with system load in the Motion Planning Rviz Display so it's might be a bit more involved than just stepping up the number.